### PR TITLE
Added user details page for admins

### PR DESCRIPTION
**Pull request by dustins**
*Issue #, if available:*

*Description of changes:*
Created a new page for administrators to manage the projects and groups a user belongs to. It is only accessible for administrators.

Recording 2024-07-12 at 16 45 07

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/constants.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export const GROUP_USERS_TABLE_NAME = 'mlspace-group-users';
 export const CONFIG_BUCKET_NAME = 'mlspace-config';
 export const DATA_BUCKET_NAME = 'mlspace-data';
 export const LOGS_BUCKET_NAME = 'mlspace-logs';
-export const ACCESS_LOGS_BUCKET_NAME = 'mlspace-access-logs';
+export const ACCESS_LOGS_BUCKET_NAME = 'mlspace-access-logs-alpha';
```
Comment (douglas1850) [inline lib/constants.ts:34]: Was this meant to be here?

File: frontend/src/modules/table/table.types.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ import { ActionCreatorWithoutPayload, AsyncThunk } from '@reduxjs/toolkit';
 type TableProps<Entry = TableEntry> = {
     tableName?: string;
     tableType?: CloudscapeTableProps.SelectionType;
-    actions?: any;
+    actions?: CallbackFunction<TableActionProps<Entry>, ReactNode | undefined>;
```
[the remark this answers is not in the document]
Comment (dustins) [reply frontend/src/modules/table/table.types.tsx:31]: I wrote PHP and javascript for many many years…types are a blessing to all mankind.

**Review by douglas1850 — APPROVED** [review]

File: frontend/src/entities/group/group.reducer.ts [hunk 3]
```diff
@@ -70,6 +70,7 @@ export const addGroupUsers = createAsyncThunk('group/add_users', async (data: Ad
 export const GroupSlice = createSlice({
     name: 'group',
     initialState,
+    reducers: {},
```
Comment (estohlmann) [inline frontend/src/entities/group/group.reducer.ts:73]: Can you tell me why this is needed, just for my education.

Comment (dustins) [reply frontend/src/entities/group/group.reducer.ts:73]: For no other reason than `createSlice` expects a `reducers` field so my linter required _something_ there.

**Review by estohlmann — APPROVED** [review]

**Pull request merged**